common/web/types/src/util/compiler-interfaces.ts

```diff
@@ -339,6 +380,28 @@ export class CompilerFileCallbacks implements CompilerCallbacks {
     return messages.find(m => failureCodes.includes(CompilerError.severity(m.code))) != undefined;
   }
 
+  /**
+   *
+   * @param event
+   * @param overrides
+   * @returns true if event has been suppressed
+   */
+  static applyMessageOverridesToEvent(event: CompilerEvent, overrides: CompilerMessageOverrideMap) {
```

Comment:
Seems this is the focal point of this PR's advertised functionality?

Reply:
yep